internal/pkg/service/buffer/worker/service/slice_close_test.go

```diff
@@ -1,333 +0,0 @@
-package service_test
-
-import (
-	"context"
-	"testing"
-	"time"
-
-	"github.com/benbjohnson/clock"
-	"github.com/keboola/go-utils/pkg/wildcards"
-	"github.com/stretchr/testify/assert"
-	etcd "go.etcd.io/etcd/client/v3"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
-	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/slicestate"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-)
-
-// TestSliceCloseTask - the worker closes the slice only after it is not used by any API node.
-func TestSliceCloseTask(t *testing.T) {
```

Comment:
Opat, je to pokryte:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31

Konkretne tu sa testuje, ze najpr musia vsetky API nodes oznami, ze sa prepli na novu slices, az tak sa odblokuje `slice.close` task:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/002_slice_upload_test.go#L41-L53